clang/include/clang/Basic/AttrDocs.td

```diff
@@ -1637,31 +1637,42 @@ specifies availability for the current target platform, the availability
 attributes are ignored. Supported platforms are:
 
 ``ios``
+``ios_app_extension``
   Apple's iOS operating system. The minimum deployment target is specified
   as part of the ``-target *arch*-apple-ios*version*`` command line argument.
   Alternatively, it can be specified by the ``-mtargetos=ios*version*``
   command-line argument.
 
+``maccatalyst``
+``maccatalyst_app_extension``
+  Apple's Mac Catalyst development environment for its macOS operating system.
+  The minimum deployment target is specified as part of the
+  ``-target *arch*-apple-ios*version*-macabi`` command line argument.
```

Comment:
All of the other descriptions include `-mtargetos`, why not here?

Reply:
I didn't know that would work for Mac Catalyst. Is it just the iOS one?

Reply:
its basically same as the platform portion of the target triple https://github.com/llvm/llvm-project/blob/main/clang/test/Driver/mtargetos-darwin.c#L4

Reply:
Aha